{"metadata":{"role":"symbol","symbolKind":"case","roleHeading":"Case","externalID":"s:5FIDO224PublicKeyCredentialErrorO18invalidCertificateyA2CmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidCertificate","kind":"identifier"}],"modules":[{"name":"FIDO2"}],"title":"PublicKeyCredentialError.invalidCertificate"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/fido2\/publickeycredentialerror\/invalidcertificate"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"invalidCertificate","kind":"identifier"}],"languages":["swift"],"platforms":["iOS"]}]}],"identifier":{"url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/invalidCertificate","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError"]]},"abstract":[{"type":"text","text":"The certifcate could not be parsed."}],"kind":"symbol","references":{"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError":{"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError","title":"PublicKeyCredentialError","type":"topic","url":"\/documentation\/fido2\/publickeycredentialerror","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"PublicKeyCredentialError"}],"role":"symbol","navigatorTitle":[{"text":"PublicKeyCredentialError","kind":"identifier"}],"abstract":[{"text":"A type that indicates when Fido2ApiClient encounters an error.","type":"text"}],"kind":"symbol"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError/invalidCertificate":{"abstract":[{"text":"The certifcate could not be parsed.","type":"text"}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/invalidCertificate","url":"\/documentation\/fido2\/publickeycredentialerror\/invalidcertificate","title":"PublicKeyCredentialError.invalidCertificate","role":"symbol","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidCertificate","kind":"identifier"}],"type":"topic"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2":{"role":"collection","type":"topic","kind":"symbol","abstract":[],"url":"\/documentation\/fido2","title":"FIDO2","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2"}}}